### PR TITLE
Add Data Source for Rebalancing Schedule

### PR DESCRIPTION
**Pull request by kasia-kujawa**

### CODE REVIEW

**Review by laimonasr — APPROVED** [review]

**Comment by aalbright-sovrn** [comment]
Hey we're looking to implement this exact feature if this could please be merged we'd love to use it

Thanks!

**Comment by kasia-kujawa** [comment]
> Hey we're looking to implement this exact feature if this could please be merged we'd love to use it
> 
> Thanks!

@aalbright-sovrn Today I plan to make small corrections so it should be merged soon :)

File: docs/data-sources/rebalancing_schedule.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "castai_rebalancing_schedule Data Source - terraform-provider-castai"
```
[the remark this answers is not in the document]
Comment (kasia-kujawa) [reply docs/data-sources/rebalancing_schedule.md:3]: Added https://github.com/castai/terraform-provider-castai/blob/79b0c7d9b90919f701fcc48937e16b75f691f2e9/examples/data-sources/castai_rebalancing_schedule/data_source.tf

File: castai/data_source_resource_rebalancing_schedule.go [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+	return &schema.Resource{
+		Description: "Retrieve Rebalancing Schedule ",
+		ReadContext: dataSourceRebalancingScheduleRead,
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Name of the Rebalancing schedule",
+			},
+			"schedule": {
```
[the remark this answers is not in the document]
Comment (kasia-kujawa) [reply castai/data_source_resource_rebalancing_schedule.go]: Changed

**Review by aldor007 — APPROVED** [review]

**Pull request merged**